it/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Concorrenza
+category: advanced
+order: 4
+lang: it
+---
+
+Uno dei punti di forza di Elixir è il suo supporto alla concorrenza. Grazie alla ErlangVM, scrivere programmi concorrenti in Elixir è più semplice di quanto uno possa aspettarsi. Il modello di concorreza si basa sugli Attori, processi indipendenti che comunicano con altri processi, attraverso il passaggio di messaggi.
+
+In questa lezione daremo un'occhiata ai moduli sulla concorrenza che vengono provveduti con l'installazione di Elixir. Nel prossimo capitolo, ci occupiamo dei _behaviour OTP_ che ci permettono di scrivere programmi concorrenti.
+
+## Tavola dei Contenuti
```

Comment:
I see "Tavola dei Contenuti" being a straight translation from "Table of Contents". I ran a quick check on a handful of Italian books (mostly technical ones) and all use "Indice" Instead. I also looked at #264 and it seems it uses "Tavola dei Contenuti" as well, so either we keep it or swap it to "Indice" everywhere. thoughts, @apeacox ?